contrib/endpoints/include/api_manager/api_manager.h

```diff
@@ -87,6 +87,12 @@ class ApiManager {
   virtual utils::Status GetStatistics(
       ApiManagerStatistics *statistics) const = 0;
 
+  // Returns the config loading status.
+  //  - Code::UNAVAILABLE Not initialized yet. The default value.
+  //  - Code::OK          Successfully initialized
+  //  - Code::ABORTED     Initialization was failed
+  virtual utils::Status ConfigLoadingStatus() = 0;
```

Comment:
The purpose of the ConfigLoadingStatus was to get access the config_loading_status_ of  the ApiManager. If we add LoadServiceRollouts, we can remove the ConfigLoadingStatus.

LoadServiceRollouts should be called only once right after the ApiManager instance was created.